resources/assets/sass/base/_base.colors.scss

```diff
@@ -0,0 +1,106 @@
+// Brand colors
+
+$brand-yellow: #f7ba00;
+$brand-yellow-dark: #d67d00;
+
+$brand-green: #78cc5c;
+$brand-green-dark: #288f18;
+
+$brand-red: #ed6262;
+$brand-red-dark: #cb4949;
+$brand-red-darker: #b13636;
+
+$brand-purple: #8b84d7;
+$brand-purple-dark: #5147c0;
+
+$brand-blue: #4ac9e3;
+$brand-blue-dark: #0087ae;
+
+$brand-orange: #ff9a3e;
+$brand-orange-dark: #d94d00;
+
+
+// Brand color map
+
+$colors: ('yellow' $brand-yellow $brand-yellow-dark),
+         ('green' $brand-green $brand-green-dark),
+         ('red' $brand-red $brand-red-dark),
+         ('purple' $brand-purple $brand-purple-dark),
+         ('blue' $brand-blue $brand-blue-dark),
+         ('orange' $brand-orange $brand-orange-dark);
```

Comment:
Color literals like `orange` should only be used in variable declarations; they should be referred to via variable everywhere else.